### PR TITLE
minor fixes

### PR DESCRIPTION
**Pull request by mssammon**
fixes bug introduced by earlier change (charOffsetToTokenIndex in TextAnnotation); TextAnnotation method to convert Curator Record to TextAnnotation used one-past-the-end label char offsets to find token indexes, but now they must be corrected first. 
Minor fixes for several issues -- #56; #57; #59; #46; possibly, #12

### CODE REVIEW

File: edison/README.md [hunk 1]
```diff
@@ -72,3 +72,27 @@ members of the class `WordFeatureExtractorFactory`.
...
+    definitions, variables, names into FeatureExtractors.
+ManifestParser actually parses the file, and stores definitions, variables, names, etc.
+
+
+## Structure of the .fex file
+
+The file has a lisp-like syntax. As in lisp, a semicolon (;) is the comment symbol. This may come anywhere
+in the line.
+
+file := (define name definition* featureslist)
```
Comment (danyaljj) [inline edison/README.md]: Maybe put these inside a triple quotes? They look very ugly ...

File: edison/README.md [hunk 2]
```diff
@@ -72,3 +72,27 @@ members of the class `WordFeatureExtractorFactory`.
 | `wordCase`                          | The word, without changing the case                                     |
 | `dateMarker`                        | An indicator for whether the token is a valid date                      |
 
+##FEX Functionality
+
+In addition to the programmatic generation using feature extractors, we allow specification of
+feature extractors through a special file that defines feature extractors. This file is the .fex file.
+
+## What happens
+FeatureManifest loads the file and sends it to ManifestParser. Contains logic for converting
```
Comment (danyaljj) [inline edison/README.md]: Class names inside quites?  FeatureManifest  => `FeatureManifest`

File: edison/README.md [hunk 3]
```diff
@@ -72,3 +72,27 @@ members of the class `WordFeatureExtractorFactory`.
 | `wordCase`                          | The word, without changing the case                                     |
 | `dateMarker`                        | An indicator for whether the token is a valid date                      |
 
+##FEX Functionality
+
+In addition to the programmatic generation using feature extractors, we allow specification of
+feature extractors through a special file that defines feature extractors. This file is the .fex file.
+
+## What happens
```
Comment (danyaljj) [inline edison/README.md]: This is subsection of "FEX functionality"; right?  So maybe triple-hash-tag?: "###"

File: edison/README.md [hunk 4]
```diff
@@ -72,3 +72,27 @@ members of the class `WordFeatureExtractorFactory`.
...
+FeatureManifest loads the file and sends it to ManifestParser. Contains logic for converting
+    definitions, variables, names into FeatureExtractors.
+ManifestParser actually parses the file, and stores definitions, variables, names, etc.
+
+
+## Structure of the .fex file
```
Comment (danyaljj) [inline edison/README.md]: Again, "###" ?

File: edison/src/test/java/edu/illinois/cs/cogcomp/edison/utilities/CreateTestTAResource.java [hunk 5]
```diff
@@ -46,7 +46,7 @@ public CreateTestTAResource() {
             // since the sentence below are not properly tokenized
             Map<String, String> nonDefaultValues = new HashMap<>();
             nonDefaultValues.put(CuratorConfigurator.RESPECT_TOKENIZATION.key,
-                    CuratorConfigurator.TRUE);
+                    CuratorConfigurator.FALSE);
```
Comment (danyaljj) [inline edison/src/test/java/edu/illinois/cs/cogcomp/edison/utilities/CreateTestTAResource.java]: So after fixing this, would it make sense to add this example which was failing somewhere in a unit test? 
https://github.com/mssammon/illinois-cogcomp-nlp-1/blob/master/edison/src/test/java/edu/illinois/cs/cogcomp/edison/utilities/CreateTestTAResource.java#L86-L87

Comment (mssammon) [inline edison/src/test/java/edu/illinois/cs/cogcomp/edison/utilities/CreateTestTAResource.java]: I don't think so: the problem arose in the TextAnnotation method that converts from Curator data structures, so the tests in the Curator module are the ones that are relevant.

**Comment by danyaljj** [comment]
Looking good. Will merge when Semaphore turns green.

**Pull request merged**